content/swiftui/concepts/search/search.md

```diff
@@ -0,0 +1,72 @@
+---
+Title: 'Search'
+Description: 'A Search Interface can be easily created using the .searchable() view modifier.'
+Subjects:
+  - 'Mobile Development'
+  - 'iOS'
+Tags:
+  - 'SwiftUI'
+  - 'Search'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+A Search Interface can be easily created using the `.searchable()` [view modifier](https://www.codecademy.com/resources/docs/swiftui/viewmodifier) applied to a `NavigationSplitView` or `NavigationStack`, or to a [view](https://www.codecademy.com/resources/docs/swiftui/views) within them.
+
+## Syntax
+
+```pseudo
+NavigationSplitView or NavigationStack
+    .searchable(
+        text: binding variable of text to search for,
+        placement: position (optional),
+        prompt: prompt text (optional)
+    )
+```
+
+Here are a few of the `placement` property types:
+
+- `.automatic` places the search box automatically, this is the default value.
+- `.sidebar` places the search box in the sidebar.
+- `.toolbar` places the search box in the toolbar.
+
```

Comment:
Can we have an explanation of what `prompt` is?

Reply:
I added in an explanation of `prompt`.